echologue-frontend/src/index.css

```
@@ -2,68 +2,274 @@
+    background: transparent;
+    color: var(--color-primary);
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+  }
+
   body {
-    @apply bg-white dark:bg-dark-bg text-gray-900 dark:text-dark-text;
-    @apply font-sans leading-relaxed;
-    @apply min-h-screen;
+    @apply font-sans leading-relaxed min-h-screen;
+    background: transparent;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)